src/messages.js

```diff
@@ -121,7 +121,6 @@ var warnings = {
   W039: "'{a}' is not allowed.",
   W040: "If a strict mode function is executed using function invocation, " +
     "its 'this' value will be undefined.",
-  W041: "Use '{a}' to compare with '{b}'.",
```

Comment:
Could you declare this property as `null` instead? Doing so is consistent with how we've handled invalid warnings in the past, and it makes it more clear to future readers that the "missing" warning is intentional.